torch/csrc/autograd/python_variable.cpp

```diff
@@ -573,6 +573,35 @@ static int THPVariable_clear(THPVariable* self) {
   return 0;
 }
 
+#if PY_VERSION_HEX >= 0x030B0000
+// Traverse and clear are required for supporting Python's GC cycle handling.
+static int THPFunction_traverse(THPFunction* self, visitproc visit, void* arg) {
```

Comment:
We expect this function to be overriden and never called. I think we want this to be empty and do
```c
static int THPFunction_traverse(THPFunction* self, visitproc visit, void* arg) {
  TORCH_INTERNAL_ASSERT(false, "Tensor tp_traverse function was not overriden properly");
  return 0;
}
```

Reply:
Right, modified as per your request.
For the track, the solution was from [python_function.cpp:L206](https://github.com/pytorch/pytorch/blob/30fb2c4abaaaa966999eab11674f25b18460e609/torch/csrc/autograd/python_function.cpp#L206)